modules/openid_connect/lib/open_project/openid_connect/hooks/hook.rb

```diff
@@ -43,20 +43,7 @@ def user_logged_in(context)
       ##
       # Once omniauth has returned with an auth hash
       # persist the access token
-      def omniauth_user_authorized(context)
-        auth_hash = context[:auth_hash]
-        controller = context[:controller]
-
-        # fetch the access token if it's present
-        access_token = auth_hash.fetch(:credentials, {})[:token]
-        # put it into a cookie
-        if controller && access_token
-          controller.send(:cookies)[:_open_project_session_access_token] = {
-            value: access_token,
-            secure: !!Rails.configuration.force_ssl
-          }
-        end
-      end
+      def omniauth_user_authorized(_context); end
```

Comment:
🟢  Q: We're retaining the empty method because of the existing API?

Reply:
Yes, it is very likely to be used in future.